Lib/test/test_mmap.py

```diff
@@ -264,6 +264,26 @@ def test_bad_file_desc(self):
         # Try opening a bad file descriptor...
         self.assertRaises(OSError, mmap.mmap, -2, 4096)
 
+    def test_if_crash(self): # test for issue gh-103987
+        with open("TESTFN", "wb+") as f:
```

Comment:
```suggestion
        with open(TESTFN, "wb+") as f:
```

Reply:
Test doesn't failed. This test affects on execution environment, so, CI/CD fails =)